pkg/render/applicationlayer/envoy-config.yaml.template

```diff
@@ -39,6 +39,12 @@
                       transport_api_version: V3
 {{- end -}}
 
+{{- define "XFF" -}}
+                xff_num_trusted_hops: {{.NumTrustedHopsXFF}}
+                use_remote_address: true
```

Comment:
did you want `use_remote_address` and `skip_xff_append` to be configurable too @electricjesus ?